### PR TITLE
{data}[intel/2017a] netcdf4-python v1.2.9, netCDF v4.4.1.1 w/ HDF5 v1.10.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/n/netCDF/netCDF-4.4.1.1-intel-2017a-HDF5-1.10.1.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+builddependencies = [
+    ('CMake', '3.8.2'),
+    ('Doxygen', '1.8.13'),
+]
+
+# make sure both static and shared libs are built
+configopts = [
+    "-DBUILD_SHARED_LIBS=OFF ",
+    "-DBUILD_SHARED_LIBS=ON ",
```
Comment (verdurin) [inline easybuild/easyconfigs/n/netCDF/netCDF-4.4.1.1-intel-2017a-HDF5-1.10.1.eb]: Is this right? Looks weird.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2695.swalot.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/0ac198719a09cadd4d3a855c33595095 for a full test report.

File: easybuild/easyconfigs/n/netCDF/netCDF-4.4.1.1-intel-2017a-HDF5-1.10.1.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/n/netCDF/netCDF-4.4.1.1-intel-2017a-HDF5-1.10.1.eb]: Yes, it basically makes EasyBuild perform the configure-build-install cycle twice: the first time with `-DBUILD_SHARED_LIBS=OFF` during configure, the 2nd time with `-DBUILD_SHARED_LIBS=ON`.

That seems to be the only way to get both shared and static libraries in the same installation...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2001.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/87b3f9ae8dee658398c923f677be8467 for a full test report.

**Comment by vanzod** [comment]
Going in, thanks @boegel!

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 9 out of 9 (2 easyconfigs in this PR)
ca003.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c7f9a30ee413c441424878d27597fa6d for a full test report.

**Pull request merged**